adal/src/main/java/com/microsoft/aad/adal/BrokerProxy.java

```diff
@@ -228,7 +228,7 @@ public AuthenticationResult getAuthTokenInBackground(final AuthenticationRequest
 
             // blocking call to get token from cache or refresh request in
             // background at Authenticator
-            AccountManagerFuture<Bundle> result = null;
+            AccountManagerFuture<Bundle> result;
```

Comment:
and made it final

---
In reply to: [65639543](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/652#discussion_r65639543) [](ancestors = 65639543)